whisk/action.go

```diff
@@ -41,6 +41,7 @@ type Action struct {
 	Error       string      `json:"error,omitempty"`
 	Code        int         `json:"code,omitempty"`
 	Publish     *bool       `json:"publish,omitempty"`
+	Updated     int64       `json:"updated"`
```

Comment:
for backwards compatibility with existing openwhisk installations i'm thinking that `json:"updated,omitempty"` should be used.

Reply:
Thank you for review. I added an `omitempty` option for backward compatibility.